doc/source/autoscaling.rst

```diff
@@ -240,6 +242,8 @@ The default idle timeout is 5 minutes. This is to prevent excessive node churn w
 Monitoring cluster status
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+The autoscaler also comes with an online dashboard. To access the dashboard, which is located on the ray head node on port ``8265``. To access it, forward the port to your local machine. You can also call ``ray dashboard`` to do this. 
```

Comment:
The wording of this is sort of weird - ray comes with a dashboard (not the autoscaler) and the dashboard isn't online.